client/js/widgets/widget.js

```diff
@@ -118,6 +118,11 @@ export class Widget extends StateManaged {
     this.applyZ();
   }
 
+  applyChildZ(child) {
```

Comment:
You're passing the child to the function, but not using it.

Reply:
Yeah it's not used in this PR but is used for the pile rendering optimization in https://github.com/ArnoldSmith86/virtualtabletop/pull/1361. If you want I can remove it from this PR and add it back there.

Reply:
Actually n/m I didn't end up directly using it there yet either - I just thought it should match the form of the other functions which notify about a child change.

Reply:
If the parameter is passed, and child.z <= this.z, then there is no need to call applyZ, since the child will not change the calculated Z value.

Reply:
Slightly more complicated, the child could have previously set `this.z` to the child's old `z` value. I've passed the previous value and done this optimization.